src/main/conversions.c

```diff
@@ -858,6 +862,8 @@ PyObject *create_py_ns_metrics_from_as_ns_metrics(as_error *error_p,
             }
         }
 
+        as_latency_release(buckets);
```

Comment:
What happens if we hit any of the error cases between the as_latency_reserve and as_latency_release? Are they properly cleaned up?

Reply:
Thanks for catching this, will fix!